packages/changelog-linker/tests/FileSystem/ChangelogFileSystem/Source/EXPECTED_CHANGELOG_LIST_DATA.md

```diff
@@ -0,0 +1,5 @@
+## Unreleased
+
+<!-- dumped content start -->
```

Comment:
I wonder, do you think this start/end is somehow useful? Maybe git diff is enough for new code highlight in IDE.

Reply:
I am not sure, do you want think it can be removed?

Reply:
I think so.
Can you give it a try?

Reply:
removed.